src/components/Modal.vue

```diff
@@ -154,6 +154,8 @@ export default createComponent({
                 const hasOpened = isOpen && !prevIsOpen;
                 const hasClosed = !isOpen && prevIsOpen;
 
+                document.body.classList.toggle("modal-is-open", isOpen);
```

Comment:
hmmm

Reply:
:thinking: 

note: I don't like this either